syft/msg/message.py

```diff
@@ -0,0 +1,38 @@
+import syft as sy
+
+from syft.workers import AbstractWorker
+
+
+class Message:
```

Comment:
Should plans inherit from message? Also this class seems like it might be a security risk, what prevents me from creating an arbitrary message?

Reply:
Not any more of a security risk than it was before, messages are still whitelisted based on the msg_type.  This is literally just a class where the tuple used to be which carried messages. Starting small.